packages/hiptext/build.sh

```diff
@@ -15,4 +15,5 @@ termux_step_pre_configure() {
 	#Because of ttf-dejavu is x11 package, the hiptext is not a x11 package.
 	install -Dm600 -t "$TERMUX_PREFIX"/share/hiptext/ \
 		"$TERMUX_PKG_SRCDIR"/DejaVuSansMono.ttf
+	autoreconf -fi
```

Comment:
Compiles fine, why `autoreconf` was needed there?
```
ar cru libhiptext.a src/libhiptext_a-artiste.o src/libhiptext_a-charquantizer.o src/libhiptext_a-font.o src/libhiptext_a-graphic.o src/libhiptext_a-hiptext.o src/libhiptext_a-jpeg.o src/libhiptext_a-macterm.o src/libhiptext_a-movie.o src/libhiptext_a-pixel.o src/libhiptext_a-pixel_parse.o src/libhiptext_a-png.o src/libhiptext_a-sixelprinter.o src/libhiptext_a-termprinter.o src/libhiptext_a-unicode.o src/libhiptext_a-xterm256.o 
ar: `u' modifier ignored since `D' is the default (see `U')
aarch64-linux-android-ranlib libhiptext.a
aarch64-linux-android-clang++  -fstack-protector-strong -Oz  -L/data/data/com.termux/files/usr/lib -Wl,-rpath=/data/data/com.termux/files/usr/lib -fopenmp -static-openmp -Wl,--enable-new-dtags -Wl,--as-needed -Wl,-z,relro,-z,now -o hiptext src/hiptext-hiptext.o libhiptext.a -ljpeg -L/data/data/com.termux/files/usr/lib -lavcodec -L/data/data/com.termux/files/usr/lib -lavformat -L/data/data/com.termux/files/usr/lib -lavutil -L/data/data/com.termux/files/usr/lib -lfreetype -L/data/data/com.termux/files/usr/lib -lgflags -L/data/data/com.termux/files/usr/lib -lglog -L/data/data/com.termux/files/usr/lib -lpng16 -lz -L/data/data/com.termux/files/usr/lib -lswscale -ljpeg 
rm -f libgtest.a
ar cru libgtest.a gtest/src/libgtest_a-gtest-all.o 
ar: `u' modifier ignored since `D' is the default (see `U')
aarch64-linux-android-ranlib libgtest.a
make[1]: Entering directory '/home/builder/.termux-build/hiptext/src'
 /usr/bin/mkdir -p '/data/data/com.termux/files/usr/bin'
  /usr/bin/install -c hiptext '/data/data/com.termux/files/usr/bin'
make[1]: Nothing to be done for 'install-data-am'.
make[1]: Leaving directory '/home/builder/.termux-build/hiptext/src'
termux - build of 'hiptext' done
```